app/models/search.rb

```diff
@@ -37,8 +37,12 @@ def single_search(params, model)
       model.find_by("result LIKE ?", params[:result])
     elsif params[:credit_card_number]
       model.find_by("credit_card_number LIKE ?", params[:credit_card_number])
+    elsif params[:updated_at]
+      model.find_by(updated_at: Time.zone.parse(params[:updated_at]))
+    elsif params[:id]
+      model.find_by(id: params[:id])
```

Comment:
this is one beastly method. after we finish everything today, let's give refactoring it a shot?

Reply:
Of course